src/Microsoft.ML.Data/Transforms/CopyColumnsTransform.cs

```diff
@@ -130,18 +109,21 @@ public sealed class Arguments : TransformInputBase
             public Column[] Column;
         }
 
-        // Factory method for SignatureLoadDataTransform.
-        private static IDataView Create(IHostEnvironment env, ModelLoadContext ctx, IDataView input)
+        /// <summary>
+        /// Factory method to create from arguments
+        /// </summary>
+        public static IDataTransform Create(IHostEnvironment env, Arguments args, IDataView input)
         {
             Contracts.CheckValue(env, nameof(env));
-            env.CheckValue(ctx, nameof(ctx));
-            ctx.CheckAtModel(GetVersionInfo());
-            env.CheckValue(input, nameof(input));
-            var transformer = Create(env, ctx);
-            return transformer.Transform(input);
+            env.CheckValue(args, nameof(args));
+
+            var transformer = new CopyColumnsTransform(env, args.Column.Select(x => (x.Source, x.Name)).ToArray());
+            return transformer.MakeDataTransform(input);
         }
 
-        // Factory method for SignatureLoadModel.
```

Comment:
> // Factory method for SignatureLoadModel. [](start = 7, length = 42)

I would prefer this format, in all other places we have this pattern instead of //summary